Dockerfile

```diff
@@ -110,33 +110,36 @@ RUN set -eux \
   && apk update --no-cache \
   && apk upgrade --no-cache \
   && apk add --no-cache bash \
-  && apk add --no-cache curl \
-  && apk add --no-cache docker \
-  && apk add --no-cache git \
-  && apk add --no-cache jq \
-  && apk add --no-cache make \
-  && apk add --no-cache ncurses \
-  && apk add --no-cache openssh \
-  && apk add --no-cache openssl \
-  && apk add --no-cache python3 \
-  && apk add --no-cache zip \
-  && if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi \
-  && python3 -m ensurepip \
-  && rm -r /usr/lib/python*/ensurepip \
-  && pip3 install --no-cache --upgrade pip setuptools wheel \
-  && if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi \
-  && python3 -m pip install ply --no-cache-dir \
-  && python3 -m pip install pyhcl --no-cache-dir \
-  && python3 -m pip install requests --no-cache-dir \
-  && python3 -m pip install slack_sdk --no-cache-dir \
-  && if [ "${AWS}" == "yes" ]; then python3 -m pip install boto3 --no-cache-dir; python3 -m pip install awscli --no-cache-dir; fi \
+	curl \
+	docker \
+	git \
+	jq \
+	make \
+	ncurses \
+	openssh \
+	openssl \
+	python3 \
+	zip \
+  ; if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi \
```

Comment:
this command would basically break python since `WORKDIR` is most likely not `/usr/bin`. This line should be removed

Reply:
Agree